From 27506249117abfa8d10bb57b5f01c91ef92ebb03 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Wed, 25 Nov 2015 13:35:22 +0000
Subject: [PATCH 6/7] spl-kmem-cache: include linux/prefetch.h for prefetchw()

This is needed for architectures that do not have a builtin prefetchw()

Signed-off-by: Dimitri John Ledkov <xnox@ubuntu.com>
Signed-off-by: Brian Behlendorf <behlendorf1@llnl.gov>
Closes #502
---
 module/spl/spl-kmem-cache.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/module/spl/spl-kmem-cache.c b/module/spl/spl-kmem-cache.c
index 2b4ac4a..7b7edfc 100644
--- a/module/spl/spl-kmem-cache.c
+++ b/module/spl/spl-kmem-cache.c
@@ -31,6 +31,7 @@
 #include <linux/swap.h>
 #include <linux/mm_compat.h>
 #include <linux/wait_compat.h>
+#include <linux/prefetch.h>
 
 /*
  * Within the scope of spl-kmem.c file the kmem_cache_* definitions
-- 
2.4.10

